src/routes/docs/products/functions/develop/+page.markdoc

```diff
@@ -694,6 +719,32 @@ If you need to send a response to the invoker of the function, such as a user, c
 The response information **will not be logged** to the Appwrite Console. 
 There are several possible ways to send a response, explore them in the following Appwrite Function.
 
+### Response Types {% #response-types %}
+
+{% table %}
+* Response
+* Description
+---
+* `send`
+* Sends a plain text body as a response.
```

Comment:
send is deprecated, right? Probably not good to mention it as one that can be used.